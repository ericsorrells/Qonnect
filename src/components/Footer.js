// ========================================================================================
import React from 'react';
import { Link }  from 'react-router-dom';
// ========================================================================================

const Footer = () => {
  return (
    <div className='footer'>
      <div className='footer__inner_container'>
        <div className='footer__title'>
          <Link to='/'>qonnect</Link>
        </div>
        <div className='footer__links'>
          <Link to='/about'>About</Link>
          <Link to='/contact'>Contact</Link>
          <Link to='/'>Blog</Link>
        </div>
        <div className='footer__social-icons'>
          <i className="fab fa-facebook"></i>
          <i className="fab fa-twitter-square"></i>
          <i className="fab fa-pinterest-square"></i>
        </div>
      </div>
    </div>
  )
}

export default Footer;