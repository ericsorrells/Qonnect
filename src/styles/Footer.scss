.footer {
  background-color: black;
  height: 300px;
  display: flex;
}

.footer__inner_container {
  width: $container;
  margin-left: auto;
  margin-right: auto;
  color: white;
}