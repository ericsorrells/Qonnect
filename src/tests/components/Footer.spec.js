// ========================================================================================
import React from 'react';
import { shallow } from 'enzyme';
import 'jest-enzyme';
import moment from 'moment';
// ========================================================================================
import Footer from '../../components/Footer';
// ========================================================================================
import { basicEvent } from '../stubs/event';
import { basicProfile } from '../stubs/profile'
// ========================================================================================

describe('<Footer />', () => {
  it('renders correctly', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});