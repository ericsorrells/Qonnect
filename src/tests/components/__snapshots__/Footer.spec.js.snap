// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="footer__inner_container"
  >
    <div
      className="footer__title"
    >
      <Link
        replace={false}
        to="/"
      >
        qonnect
      </Link>
    </div>
    <div
      className="footer__links"
    >
      <Link
        replace={false}
        to="/about"
      >
        About
      </Link>
      <Link
        replace={false}
        to="/contact"
      >
        Contact
      </Link>
      <Link
        replace={false}
        to="/"
      >
        Blog
      </Link>
    </div>
    <div
      className="footer__social-icons"
    >
      <i
        className="fab fa-facebook"
      />
      <i
        className="fab fa-twitter-square"
      />
      <i
        className="fab fa-pinterest-square"
      />
    </div>
  </div>
</div>
`;
